source/authentication/anonymous.txt

```diff
@@ -113,6 +113,8 @@ Configuration
              "disabled": <boolean>
            }
          }
+      
```

Comment:
nit: preceding code-block language is defined as `javascript` here, but is `none` in email-password.txt and api-key.txt. Can we update so all 3 match?